stellar_account_prometheus_exporter/exporter.py

```diff
@@ -83,11 +87,19 @@ def do_GET(self):
                 if "balances" not in r.json():
                     self.error(500, "Error - no balances found for account {}".format(account["account_id"]))
                     return
+                if "num_sponsored" not in r.json():
+                    self.error(500, "Error - no num_sponsored found for account {}".format(account["account_id"]))
+                    return
+                if "num_sponsoring" not in r.json():
+                    self.error(500, "Error - no num_sponsoring found for account {}".format(account["account_id"]))
+                    return
                 for balance in r.json()["balances"]:
                     labels = [network["name"], account["account_id"], account["account_name"], balance["asset_type"]]
                     m_balance.labels(*labels).set(balance["balance"])
                     m_buying_liabilities.labels(*labels).set(balance["buying_liabilities"])
                     m_selling_liabilities.labels(*labels).set(balance["selling_liabilities"])
+                    m_num_sponsored.labels(*labels).set(r.json()["num_sponsored"])
+                    m_num_sponsoring.labels(*labels).set(r.json()["num_sponsoring"])
```

Comment:
I think those two should be unindented one level. `num_sponsored` and `num_sponsoring` values are not per balance but per account so we should only export them once per account